operator/controllers/suite_test.go

```diff
@@ -76,14 +77,22 @@ var _ = BeforeSuite(func() {
 
 	manifestCrd := &v1.CustomResourceDefinition{}
 	res, err := http.DefaultClient.Get(
-		"https://raw.githubusercontent.com/adityabhatia/manifest-operator/overrideApiChanges/api/config/crd/bases/component.kyma-project.io_manifests.yaml") //nolint:lll
+		"https://raw.githubusercontent.com/kyma-project/manifest-operator/main/operator/config/crd/bases/component.kyma-project.io_manifests.yaml") //nolint:lll
```

Comment:
To Reviewer: the manifests crd is based on the latest main, some fields has been updated.